pkg/query-service/model/response.go

```diff
@@ -269,6 +269,32 @@ type SearchSpanResponseItem struct {
 	SpanKind         string            `json:"spanKind"`
 }
 
+type SearchSpanResponseItemV2 struct {
+	TimeUnixNano      time.Time          `json:"timestamp" ch:"timestamp"`
+	DurationNano      uint64             `json:"durationNano" ch:"duration_nano"`
+	SpanID            string             `json:"spanId" ch:"span_id"`
+	TraceID           string             `json:"traceId" ch:"trace_id"`
+	HasError          bool               `json:"hasError" ch:"has_error"`
+	Kind              int8               `json:"kind" ch:"kind"`
+	ServiceName       string             `json:"serviceName" ch:"resource_string_service$$name"`
+	Name              string             `json:"name" ch:"name"`
+	References        string             `json:"references,omitempty" ch:"references"`
+	Attributes_string map[string]string  `json:"attributes_string" ch:"attributes_string"`
+	Attributes_number map[string]float64 `json:"attributes_number" ch:"attributes_number"`
+	Attributes_bool   map[string]bool    `json:"attributes_bool" ch:"attributes_bool"`
```

Comment:
I do not think attributes should be separated by datatype in this API response. Why not return the original data with map[string]any?

Reply:
Correct, I am doing that in the the function to keep the actual response same https://github.com/SigNoz/signoz/blob/4f89f7e6fc26985566c8271355a45df8f376883b/pkg/query-service/app/clickhouseReader/reader.go#L1167